pkg/didcomm/transport/transport_interface.go

```diff
@@ -12,3 +12,6 @@ type OutboundTransport interface {
 	// Send send a2a exchange data
 	Send(data string, destination string) (string, error)
 }
+
+// for Inbound Transport, use of dynamic handler is sufficient, no interface is needed
```

Comment:
@Baha-sk this comment is out of place. godocs should be attached to a package or a type

Reply:
removed comment